### PR TITLE
add garden finance

### PR DESCRIPTION
**Pull request by Nesopie**
Bridge adapter for garden.finance. Bitcoin to Ethereum trades are tracked through deposits and withdrawals on Ethereum and similarly for Bitcoin to Arbitrum trades. There are no trades between Ethereum and Arbitrum

### CODE REVIEW

File: src/data/bridgeNetworkData.ts [hunk 1]
```diff
@@ -738,6 +738,27 @@ export default [
...
+    url: "",
+    chains: ["Ethereum", "Polygon", "BSC", "MAP Relay Chain"],
+    chainMapping: {
+      "map relay chain": "map",
+    },
+    destinationChain: "MAP Relay Chain",
+  },
+  {
+    id: 48,
+    displayName: "Garden Finance",
```
Comment (vrtnd) [inline src/data/bridgeNetworkData.ts:756]: u need to list your protocl on defillama first

Comment (Nesopie) [reply src/data/bridgeNetworkData.ts:756]: [You mean this right?](https://docs.llama.fi/list-your-project/submit-a-project)? TVL currently doesn't make sense for our project so we haven't written a TVL adapter. Is there any other way to list the project?

Comment (vrtnd) [reply src/data/bridgeNetworkData.ts:756]: yes but tvl isn't required, u should at least provide logo, twiiter handle etc

Comment (Nesopie) [reply src/data/bridgeNetworkData.ts:756]: I see. And this is done on the [defillama-server](https://github.com/DefiLlama/defillama-server/tree/master/defi/src/protocols) repo?

Comment (Nesopie) [reply src/data/bridgeNetworkData.ts]: Hey! It's been listed

**Pull request merged**